src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -192,11 +197,22 @@ function (Document $document) use ($dbForConsole, $getProjectDB) {
                     return;
                 }
 
-                $function = $getProjectDB($project)->getDocument('functions', $document->getAttribute('resourceId'));
+                $resource = $getProjectDB($project)->getDocument(
+                    $document->getAttribute('resourceCollection'),
+                    $document->getAttribute('resourceId')
```

Comment:
Should we use `resourceInternalId` instead of `resourceId`?

Reply:
We can't call `getDocument` with an internal ID